src/components/institution/InstitutionAutocomplete.tsx

```diff
@@ -42,7 +45,7 @@ const InstitutionAutocomplete: FC<InstitutionAutocompleteProps> = ({
       renderInput={(params) => (
         <TextField
           {...params}
-          label={t('institution')}
+          label={readOnlyValue ?? t('institution')}
```

Comment:
skal ikke `readOnlyValue` inn som value og `disable` settes til true?

Reply:
hvis vi setter value, så blir det en controlled component, og får da en feilmelding når vi endrer til å være uncontrolled

Reply:
virker som en hack å sette verdien som label i stedet for da. når går den over til å bli uncontrolled? jeg trodde egentlig denne alltid var controlled

Reply:
fant ut at `inputValue` var det rette å bruke her 👍

Reply:
![Skjermbilde 2020-06-03 kl  09 38 25](https://user-images.githubusercontent.com/6595982/83609061-036a9380-a57e-11ea-9414-52f4949c7911.png)

...men får fortsatt feilmelding når jeg går fra å endre en institusjon til å opprette en institusjon

Reply:
får feilmeldingen i det jeg velger en institusjon

Reply:
hmm, mangler vi noe initialValues da? vi setter sikkert value til `undefined` en plass, når vi egentlig vil sette `null` eller tom streng...

Reply:
det blir kanskje trøbbel siden vi ikke vet hvor mange nivå det er? slik at vi bare setter initialValue for toppnivå?

Reply:
ja, det kan være. skal sjekke litt mer

Reply:
det var formik sin `setValues` som autocompleten ikke likte tydeligvis. Endret til `setFieldValue`, og da gikk det fint.